src/components/ReceiptImage.tsx

```diff
@@ -53,6 +53,9 @@ type ReceiptImageProps = (
     /** Whether we should display the receipt with ThumbnailImage component */
     shouldUseThumbnailImage?: boolean;
 
+    /** Whether we should display the receipt with inital object position  */
```

Comment:
NAB - This isn't immediately clear to me what this means - is the "initial object position" the center of the image? Or can it be something else even?

```suggestion
    /** Whether we should display the receipt with initial object position */
```

Reply:
Hmm making this NAB because I see that "initial" gets passed all the way down to the react native image component... So this isn't something we necessarily need to clarify in this PR